pkg/agent/agent.go

```diff
@@ -168,7 +169,7 @@ func (i *Initializer) GetWireGuardClient() wireguard.Interface {
 // setupOVSBridge sets up the OVS bridge and create host gateway interface and tunnel port
 func (i *Initializer) setupOVSBridge() error {
 	if err := i.ovsBridgeClient.Create(); err != nil {
-		klog.Error("Failed to create OVS bridge: ", err)
+		klog.Errorf("Failed to create OVS bridge")
```

Comment:
Use klog.ErrorS(err, "Failed to create OVS bridge")

Reply:
Done